document_session_cluster_transaction.go

```diff
@@ -0,0 +1,328 @@
+package ravendb
+
+import (
+	"reflect"
+)
+
+type ClusterTransactionOperations struct {
+	session                             *InMemoryDocumentSessionOperations
+	state                               map[string]*CompareExchangeSessionValue
```

Comment:
Aren't we missing `_compareExchangeIncludes` ?

Reply:
We are, but we don't have any builder API for load so I decided not to write it from scratch in this PR, I've created a ticket for it: https://issues.hibernatingrhinos.com/issue/RDBC-786/Include-CompareExchangeValue-in-Load-method-in-go-client